src/vs/workbench/contrib/debug/browser/welcomeView.ts

```diff
@@ -70,14 +80,26 @@ export class WelcomeView extends ViewPane {
 			if (isCodeEditor(editorControl)) {
 				const model = editorControl.getModel();
 				const language = model ? model.getLanguageId() : undefined;
-				if (language && this.debugService.getAdapterManager().someDebuggerInterestedInLanguage(language)) {
+				// --- Start Positron ---
+				// if (language && this.debugService.getAdapterManager().someDebuggerInterestedInLanguage(language)) {
+				// Find debuggers for this language and check if any support UI launch
+				const adapterManager = this.debugService.getAdapterManager();
+				if (language && adapterManager.someDebuggerInterestedInLanguage(language)) {
 					this.debugStartLanguageContext.set(language);
 					this.debuggerInterestedContext.set(true);
 					storageSevice.store(debugStartLanguageKey, language, StorageScope.WORKSPACE, StorageTarget.MACHINE);
+
```

Comment:
It would help to have this original code fenced

```suggestion
				if (language && adapterManager.someDebuggerInterestedInLanguage(language)) {
				    // --- End Positron ---
					this.debugStartLanguageContext.set(language);
					this.debuggerInterestedContext.set(true);
					storageSevice.store(debugStartLanguageKey, language, StorageScope.WORKSPACE, StorageTarget.MACHINE);
					// --- Start Positron ---

```